pkg/config/types/generated_constants.go

```diff
@@ -1,124 +1,124 @@
+const JobDefaultsServiceTaskResourcesMemoryKey = "jobdefaults.service.task.resources.memory"
 const LoggingLevelKey = "logging.level"
-const LoggingModeKey = "logging.mode"
 const LoggingLogDebugInfoIntervalKey = "logging.logdebuginfointerval"
+const LoggingModeKey = "logging.mode"
+const NameProviderKey = "nameprovider"
+const OrchestratorAdvertiseKey = "orchestrator.advertise"
+const OrchestratorAuthSecretKey = "orchestrator.authsecret"
+const OrchestratorClusterAdvertiseKey = "orchestrator.cluster.advertise"
+const OrchestratorClusterHostKey = "orchestrator.cluster.host"
+const OrchestratorClusterNameKey = "orchestrator.cluster.name"
+const OrchestratorClusterPeersKey = "orchestrator.cluster.peers"
+const OrchestratorClusterPortKey = "orchestrator.cluster.port"
+const OrchestratorEnabledKey = "orchestrator.enabled"
+const OrchestratorEvaluationBrokerMaxRetryCountKey = "orchestrator.evaluationbroker.maxretrycount"
+const OrchestratorEvaluationBrokerVisibilityTimeoutKey = "orchestrator.evaluationbroker.visibilitytimeout"
+const OrchestratorHostKey = "orchestrator.host"
+const OrchestratorNodeManagerDisconnectTimeoutKey = "orchestrator.nodemanager.disconnecttimeout"
+const OrchestratorNodeManagerManualApprovalKey = "orchestrator.nodemanager.manualapproval"
+const OrchestratorPortKey = "orchestrator.port"
+const OrchestratorSchedulerHousekeepingIntervalKey = "orchestrator.scheduler.housekeepinginterval"
+const OrchestratorSchedulerHousekeepingTimeoutKey = "orchestrator.scheduler.housekeepingtimeout"
+const OrchestratorSchedulerWorkerCountKey = "orchestrator.scheduler.workercount"
+const OrchestratorTLSAutoCertKey = "orchestrator.tls.autocert"
+const OrchestratorTLSAutoCertCachePathKey = "orchestrator.tls.autocertcachepath"
+const OrchestratorTLSCAFileKey = "orchestrator.tls.cafile"
+const OrchestratorTLSCertFileKey = "orchestrator.tls.certfile"
+const OrchestratorTLSInsecureKey = "orchestrator.tls.insecure"
+const OrchestratorTLSKeyFileKey = "orchestrator.tls.keyfile"
+const OrchestratorTLSSelfSignedKey = "orchestrator.tls.selfsigned"
+const OrchestratorTLSUseTLSKey = "orchestrator.tls.usetls"
+const PublishersDisabledKey = "publishers.disabled"
+const PublishersTypesIPFSEndpointKey = "publishers.types.ipfs.endpoint"
+const PublishersTypesLocalAddressKey = "publishers.types.local.address"
+const PublishersTypesLocalDirectoryKey = "publishers.types.local.directory"
+const PublishersTypesLocalPortKey = "publishers.types.local.port"
+const PublishersTypesS3PreSignedURLDisabledKey = "publishers.types.s3.presignedurldisabled"
+const PublishersTypesS3PreSignedURLExpirationKey = "publishers.types.s3.presignedurlexpiration"
+const ResultDownloadersDisabledKey = "resultdownloaders.disabled"
+const ResultDownloadersTimeoutKey = "resultdownloaders.timeout"
+const ResultDownloadersTypesIPFSEndpointKey = "resultdownloaders.types.ipfs.endpoint"
+const StrictVersionMatchKey = "strictversionmatch"
```

Comment:
is this used?

Reply:
The key? Yes, its used for validation. 
The actual config value? I believe the requester uses it some how, but not sure off the top of my head. This value existed before the PR, the diff here is due to sorting fields alphabetically.